src/core/ipv4/ip4_napt.c

```diff
@@ -39,9 +39,11 @@
  *
  */
 
-#if ESP_LWIP
-#if LWIP_IPV4
-#if IP_NAPT
+#if ESP_LWIP && LWIP_IPV4 && IP_NAPT
+
+#include "assert.h"
+#include "stdbool.h"
+#include "string.h"
```

Comment:
Nitpick (same above)
```suggestion
#include <string.h>
```